app/scripts/views/settings/clients.js

```diff
@@ -170,6 +202,17 @@ define(function (require, exports, module) {
             }
           });
       }
+    },
+
+    _fetchApps: function () {
```

Comment:
Again, `App` is overloaded now, if this is an OAuth app, please add a prefix for clarity.